processes/issue-triage.md

```diff
@@ -0,0 +1,95 @@
+# Issue Triage
+
+## What is triaging?
+Triaging is a process where maintainers review, respond to, and organize GitHub issues and pull requests. Triaging involves categorizing issues and pull requests based on factors such as kind (bug, enhancement, etc.) and priority/urgency. At the end of this process, new issues will be categorized appropriately and team leaders will be able to make backlog prioritization decisions, as needed.
+
+## Why is triaging important?
+Triaging is important because it:
+- promotes a responsive and inclusive community
+- leads to greater transparency, better discussions, and more collaborative, informed decision-making
+- speeds up issue management
+- prevents work from lingering endlessly
+- helps build prioritization, negotiation, and decision-making skills, which are critical to most tech roles.
+
+## Who is responsible?
+The responsibility is open to anyone. If no one volunteers to triage issues, it falls on the maintainers of each tool/repo. Maintainers are listed in the `MAINTAINERS.md` file of each repo.
```

Comment:
@microwavables I believe that you had a concern about maintaining the `MAINTAINERS.md` file in each repo. My reasoning for this is so that it would be easier to track when you're in a specific repo. For example, if someone wanted to see the maintainers of the ytt repo (`carvel-ytt`), _I think_ it would be more intuitive to look at `carvel-ytt/MAINTAINERS.md` than in `carvel/MAINTAINERS.md`.

Reply:
@aaronshurley I was assuming that we would create a section in the `README.md` in each repo that would contain links to `carvel/MAINTAINERS.md`; `carvel/GOVERNANCE.md`; `carvel/CONTRIBUTING.md`; `carvel/CODE-OF-CONDUCT.md`; the triage process doc, backlog management, and contributor model. That way we have one single source of truth to maintain/update. Within the `carvel/MAINTAINERS.md` we can specify which tool each maintainer is responsible for and on what level (maintainer/approver/reviewer). wdyt?

Reply:
@microwavables I see what you're saying. It looks like we're making different assumptions :)

_I was assuming_ that we'd have "placeholder" versions of those documents in each repo and those docs would have a link pointing to the `carvel/` source. However, `MAINTAINERS.md` is an interesting one because it will vary from repo to repo, unlike those other documents. This is the primary reason why I think it's important to have in the repo itself.

On another note, I didn't think we'd need to add anything to the README.md section since these types of files are fairly standard but I'm open to that.

Reply:
As a follow-up from a [slack conversation](https://kubernetes.slack.com/archives/CH8KCCKA5/p1627408494147300), it sounds like the team prefers to have a single file in `carvel`. I'll update this wording.